src/components/index/StatisticsDisplay.tsx

```diff
@@ -0,0 +1,144 @@
+import { Container } from "@components/commons/Container";
+import { useUnitSuffix } from "@hooks/useUnitSuffix";
+import {
+  StatsData,
+  SupplyData,
+} from "@defichain/whale-api-client/dist/api/stats";
+import { useEffect, useRef, useState } from "react";
+import CountUp from "react-countup";
+import classNames from "classnames";
+import { useTranslation } from "next-i18next";
+import { useWindowDimensions } from "@hooks/useWindowDimensions";
+import { useWhaleApiClient } from "../../layouts/context/WhaleContext";
+import { calculatePercentage } from "../../shared/calculatePercentage";
+
+export function StatsDisplay() {
+  const api = useWhaleApiClient();
+  const [stats, setStats] = useState<StatsData>();
+  const [supply, setSupply] = useState<SupplyData>();
+  const { t } = useTranslation("page-index");
+  const isFirstLoad = useRef(true);
+  const dimensions = useWindowDimensions();
+
+  useEffect(() => {
+    if (isFirstLoad.current) {
+      isFirstLoad.current = false;
+      void api.stats.getSupply().then((supplyItem) => {
+        setSupply(supplyItem);
+      });
+
+      void api.stats.get().then((statsItem) => {
+        setStats(statsItem);
+      });
+    }
+  });
+
+  const { suffix, value } = useUnitSuffix(
+    stats ? stats.tvl.masternodes.toString() : "N/A"
+  );
+  const masternodeValue = `${value + suffix}+`;
+
+  return (
+    <div className="relative z-0">
+      <Container className="relative lg:mt-[57px] mt-[76px]">
+        <div
+          id="statistics_display"
+          className="scroll-mt-[200px] card-outline-2 p-[0.5px] rounded-[30px]"
+        >
+          <div className="card-bg rounded-[30px] py-10 px-6">
+            <div className="flex lg:flex-row flex-col gap-y-4">
+              <StatsItem
+                title={t("StatisticsSection.dfiMinted.title")}
+                stats={supply?.total ?? undefined}
```

Comment:
your changes means if it is undefined, return undefined

```suggestion
                stats={supply?.total}
```